Dev prav

pravrai opened the pull request.

pravrai committed:
Commit changes

Added logger logging
Created a static class that holds static types, added them to
Assets.Scripts.Utility namespace

pravrai committed:
added comments

review-ninja commented:
<a href="https://app.review.ninja/FrozenMetalStudios/Soulless/pull/7" target="_blank"><img src="https://app.review.ninja/45891207/pull/7/badge" alt="ReviewNinja"/>

pravrai committed:
Commit to resolve conflicts

pravrai committed:
Clean up

After conflict resolving clean up, my scene required some touch up to
work again.
Changed bracketing from
class name{
to
class name
{

DanApperloo committed:
Actually resolve merge for Prav
